OpenUpSA/wazimap-ng
# Added a dropdown for choropleth range selection

## Description
Added a new field in the profile indicator page to configure the choropleth range type

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-440

## How to test it locally
* go to any profile indicator page
* change the choropleth range type using the new dropdown
* on the ui confirm that the dropdown legend values are calculated accordingly

## Changelog

### Added

### Updated
* `wazimap_ng/profile/admin/admins/profile_indicator_admin.py` to add field
* `wazimap_ng/profile/models.py` to modify the ProfileIndicator model

### Removed


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
